dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/AssemblyCheckFilterProcessor.java

```diff
@@ -45,7 +45,14 @@ public AssemblyCheckFilterProcessor(FastaSequenceReader fastaSequenceReader) {
      */
     @Override
     public SubSnpCoreFields process(SubSnpCoreFields subSnpCoreFields) throws Exception {
-        String referenceAllele = subSnpCoreFields.getReferenceInForwardStrand();
+        String referenceAllele;
+        try {
+            referenceAllele = subSnpCoreFields.getReferenceInForwardStrand();
+        } catch (IllegalArgumentException hgvsReferenceUndefined) {
+            logger.debug("Variant filtered out because reference allele is not defined: {} ({})", subSnpCoreFields,
+                         hgvsReferenceUndefined);
```

Comment:
does having hgvsReferenceUndefined as an argument automatically call hgvsReferenceUndefined.getMessage()?

Reply:
the logger method takes `Object... object` which will replace the `{}` calling `object.toString()`. the `Throwable.toString()` does call the getMessage method.